showcase/tabular/wide_and_deep.py

```diff
@@ -0,0 +1,145 @@
+# Copyright 2019 The FastEstimator Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+import tensorflow as tf
+import numpy as np
+from tensorflow.python.keras.layers import Input, Embedding, Dense, Flatten, concatenate
+from tensorflow.python.keras.models import Model
+
+from sklearn.preprocessing import StandardScaler
+
+from fastestimator.estimator.estimator import Estimator
+from fastestimator.pipeline.pipeline import Pipeline
+from fastestimator.network.network import Network
+from fastestimator.network.model import ModelOp, build
+from fastestimator.network.loss import BinaryCrossentropy
+from fastestimator.estimator.trace import Accuracy
+from fastestimator.dataset import movielens20m_data
+
+_USER_INPUT_DIM = 138493
+_MOVIE_INPUT_DIM = 26744
+_USER_EMBEDDING_DIM = 64
+_MOVIE_EMBEDDING_DIM = 15
+
+GENRES = [
```

Comment:
the should probably be in the data file

Reply:
Sure. This makes sense. I will move this to the data file.